### PR TITLE
Vmo 2631 fix storyshots and add automated regression test on GitHub

### PR DESCRIPTION
**Pull request by safydy-r**
[Summary]
- we [configured travis ci](https://github.com/FLOIP/flow-builder/pull/18/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R6) to call this command `yarn test:unit --no-cache` , which triggers the storyshot from `tests/unit/storybook.spec.ts`
- we have a choice to choose when to build? (Push AND/OR Pull request), [check details](https://docs.travis-ci.com/user/web-ui/#build-pushed-branches). I suggest we only use PR build, that way we will have builds for new PR + for new pushes if a PR is still open (see small status icon before commit hash for my last few pushes on this PR)
- travis CI will run build in my travis-ci account, last status details is here : https://travis-ci.com/github/FLOIP/flow-builder

### CODE REVIEW

File: src/store/index.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import {StoreOptions} from 'vuex'
 import {IFlowsState, store as flow} from './flow'
 import {IBuilderState, store as builder} from './builder'
-import trees from './trees/trees'
+// @ts-ignore
+import trees from '@/store/trees/trees.js'
```
Comment (jcbashdown) [inline src/store/index.ts:5]: Can we try and avoid the ts-ignore? We should try not to introduce new ones

**Review by jcbashdown — COMMENTED** [review]
Lets add a status badge to the readme:

https://docs.travis-ci.com/user/status-images/

~~I wonder if we can add them to all PRs automatically somehow..?~~ - This is already working

**Comment by safydy-r** [comment]
TODO: Will check travis-ci plan & credit usage

**Comment by safydy-r** [comment]
[Note from travis-ci about credits: https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing]

We love our OSS teams who choose to build and test using TravisCI and we fully want to support that community. However, in recent months we have encountered significant abuse of the intention of this offering (increased activity of cryptocurrency miners, TOR nodes operators etc.). Abusers have been tying up our build queues and causing performance reductions for everyone. In order to bring the rules back to fair playing grounds, we are implementing some changes for our public build repositories.

    For those of you who have been building on public repositories (on travis-ci.com, with no paid subscription), we will upgrade you to our trial (free) plan with a 10K credit allotment (which allows around 1000 minutes in a Linux environment).
    You will not need to change your build definitions when you are pointed to the new plan
    When your credit allotment runs out - we’d love for you to consider which of our plans will meet your needs.
    We will be offering an allotment of OSS minutes that will be reviewed and allocated on a case by case basis. Should you want to apply for these credits please open a request with Travis CI support stating that you’d like to be considered for the OSS allotment. Please include:
        Your account name and VCS provider (like travis-ci.com/github/[your account name] )
        How many credits (build minutes) you’d like to request (should your run out of credits again you can repeat the process to request more or discuss a renewable amount)
    Usage will be tracked under your account information so that you can better understand how many credits/minutes are being used

**Comment by safydy-r** [comment]
@jcbashdown 
> Lets add a status badge to the readme:
> 
> https://docs.travis-ci.com/user/status-images/

I have added it: https://github.com/FLOIP/flow-builder/pull/18/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1

File: src/store/index.ts (hunk 1 above)
Comment (safydy-r) [reply src/store/index.ts:5]: What do you mean here exactly @jcbashdown ?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.com/FLOIP/flow-builder.svg?branch=master)](https://travis-ci.com/FLOIP/flow-builder)
```
Comment (jcbashdown) [inline README.md:1]: Nice! Assuming it will not longer appear as `build: unknown` when we merge

**Review by jcbashdown — APPROVED** [review]

**Pull request merged**